File: src/msa/web-service/portalapi/src/main/java/vn/com/kns/portalapi/application/service/administration/auditLog/dto/AuditLogDto.java
package vn.com.kns.portalapi.application.service.administration.auditLog.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuditLogDto {
    private Long id;
    private Date createdDate;
    private String createdBy;
    private Long execDuration;
    private String clientIpAddress;
    private String clientName;
    private String browserInfo;
    private String path;
    private String httpStatus;
    private String exception;
    private String input;
    private String output;
}

File: src/msa/main-service/authw-service/src/app/shared/services/app/audit-log/dto/audit-log-dto.ts
export class AuditLogDto {
  id: number | null;
  createdDate: Date;
  createdBy: string;
  execDuration: number;
  clientIpAddress: string;
  clientName: string;
  browserInfo: string;
  path: string;
  httpStatus: string;
  exception: string;
  input: string;
  output: string;
}
